code/game/objects/items/weapons/autolathe_disk/guild_disks.dm

```diff
@@ -120,7 +120,7 @@
 		/datum/design/autolathe/conveyor = 0,
 		/datum/design/autolathe/conveyor_switch = 0,
 		///datum/design/autolathe/circuit/smelter = 3, //Balance, no more rnd/guild abuse
-		/datum/design/autolathe/circuit/sorter
+		/datum/design/autolathe/circuit/sorter = 0
```

Comment:
```suggestion
		/datum/design/autolathe/circuit/sorter = 3
```

Reply:
Why?

Reply:
balance, we had Sci making soters and do mining to be self suffusion
We also had guild/props doing the same to the church